src/routes/safe/components/Apps/ListContentLayout/Layout.jsx

```diff
@@ -0,0 +1,46 @@
+// @flow
+import styled from 'styled-components'
+
+export const Wrapper = styled.div`
+  display: grid;
+  grid-template-columns: 245px auto;
+  grid-template-rows: 62px auto 25px;
+  min-height: 500px;
+
+  .background {
+    box-shadow: 1px 2px 10px 0 rgba(212, 212, 211, 0.59);
+    background-color: white;
+  }
+`
+export const Nav = styled.div`
+  grid-column: 1/3;
+  grid-row: 1;
+  margin: 8px 0;
+  padding: 16px 0;
+  box-sizing: border-box;
+  display: flex;
+  justify-content: flex-end;
+`
+
+export const Menu = styled.div.attrs(() => ({ className: 'background' }))`
```

Comment:
what is this classname used for?

Reply:
that provides the styles for the background, defined in line 10.